lib/src/maintenance.dart

```diff
@@ -151,8 +151,7 @@ Future<Maintenance> detectMaintenance(
           final analyzer = map['analyzer'];
           if (analyzer != null) {
             final value = analyzer['strong-mode'];
-            strongModeEnabled =
-                value != null && (value == true || value is Map);
+            strongModeDisabled = value is bool && value == false;
```

Comment:
`dartanalyzer` emits a hint to remove it if you have `strong-mode: false`. As we override the options file, we'll need to check for it manually. I don't think it will effect too much package though...